src/mprismanager.cpp

```diff
@@ -154,6 +163,35 @@ void MprisManager::setSingleService(bool single)
     emit singleServiceChanged();
 }
 
+bool MprisManager::mprisBluezProxyEnabled() const
+{
+    return m_mprisBluezProxyEnabled;
+}
+
+void MprisManager::setMprisBluezProxyEnabled(bool enabled)
+{
+    if (m_mprisBluezProxyEnabled == enabled) {
+        return;
+    }
+
+    if (enabled) {
+        // Create MprisBluezProxy instance
+        qmlInfo(this) << "Enabling MprisBluezProxy";
+        if (m_mprisBluezProxy.isNull()) {
+            m_mprisBluezProxy = QSharedPointer<MprisBluezProxy>(new MprisBluezProxy(this));
```

Comment:
Not sure if this either needs QSharedPointer when it's already parented.
For small thing the m_mprisBluezProxyEnabled member is also redundant, the same as pointer not null.